app/models/loan.rb

```diff
@@ -3,4 +3,19 @@ class Loan < ActiveRecord::Base
   belongs_to :order
   validates :order_id, presence: true
   validates :project_id, presence: true
+  validates :amount, presence: true, numericality:
+    {
+      greater_than_or_equal_to: 1000,
+      less_than_or_equal_to: :project_amount_needed
+    }
+
+  private
+
+  def project_amount_needed
+    if project.nil?
+      1000
```

Comment:
Because of the way the validation works, I needed the less_than_or_equal_to to be larger than the greater_than_or_equal_to.